lte/gateway/python/integ_tests/federated_tests/README.md

```diff
@@ -0,0 +1,95 @@
+---
+id: federated_integ_test
+title: Federated Integration Tests
+hide_title: true
+---
+# Federated Integration Tests (EXPERIMENTAL)
+
+The objective of Federated Integration Tests is to provide a test platform
+where to run Magma with all its components.
+
+As the diagram indicates, this test spins up AGW, FEG and Orc8r
+and uses S1aP and trf server to run tests.
+
+```mermaid
+graph LR;
+    S1AP --> AGW --> orc8r --> FEG --> 3ggp_CORE;
+    3ggp_CORE --> FEG --> orc8r --> AGW --> S1AP;
+    AGW --> trf_server;
+    trf_server --> AGW;
+```
+
+The processes will run on the following machines:
+- AGW will run on **magma** vm.
+- FEG and Orc8r will run on our **local docker**.
+- Traffic server will run on **magma_trf** vm
+- Test will be started from **magma_test** vm
+
+## Build environment
+### Automatic build
+On your host machine go to
+`cd  magma/lte/gateway/python/integ_tests/federated_tests` and run
+```
+fab build_all
+```
+Once it is built, start the traffic and test vm
+```
+cd magma/lte/gateway
+vagrant up magma_test
+vagrant up magma_trf
+```
+
+### Manual build
+If you want to build the environment manually:
+- AGW:
+```
+cd magma/lte/gateway
+vagrant up magma
+vagrant ssh magma
+# inside vagrant vm
+cd magma/lte/gateway
+make run
+```
+- FEG:
+```
+cd magma/lte/gateway/python/integ_tests/federated_tests/docker
```

Comment:
You mentioned that feg is run on the host machine. but I think people will still be confused, so it might be good to say here again to run this on the host machine.

Reply:
Thanks

I added a comment and include the command 

```
# exit from vagrant vm
exit
```

to all the actions done on AGW